enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationStatusService.java

```diff
@@ -255,6 +259,7 @@ private void evaluatePresenceCheckResult(OwnerId ownerId,
                 break;
             case REJECTED:
                 idVerification.setRejectReason(result.getRejectReason());
+                idVerification.setErrorOrigin(ErrorOrigin.PRESENCE_CHECK);
```

Comment:
Maybe we should consider introducing `rejectOrigin` to not mix failed error and rejected reason together on the `errorOrigin` field.